### ISSUE TITLE
Hydra 0.13  Reload config  error message

### ISSUE THREAD

**Issue by tinylamb** [sdegutis/mjolnir#278]
I put following config file into my ~/.hydra/ .
dotfiles / home / .hydra / grid.lua  and init.lua

I have checked the the accessibility,but failed to reload it.The runtime error message is:
attempt to index field 'autolaunch'(a nil value).

How to fix it?

**Comment by sdegutis** [comment]
@tinylamb My config (the one you're using) has been updated to use the latest API on the master branch, which is different than the API for version 0.13, sorry about that. I expect a release any day now which should clear that up.
